### ISSUE TITLE
Complete back transformation from EO to bytecode

### ISSUE THREAD

**Issue by volodya-lombrozo** [objectionary/jeo-maven-plugin#108]
Right now we just compile bytecode from `listing` element of EO (`xmir`). So it's actually not a transformation - we just copy bytecode, save it into `listing` and then just create bytecode from that listing:
```java
@Override
public byte[] toBytecode() {
  return Base64.getDecoder().decode(this.xml.xpath("/program/listing/text()").get(0));
}
```
By doing this we actually ignore the results created by `BytecodeRepresentation#toEO`, which is wrong, of course. Moreover it keeps open the question about the format of EO representation and blocks the next issues: #102, #93, #60.

Hence, we have to create bytecode honestly from the entire EO representation without using `listing`.
Entry point for the implementation is `EoRepresentation#toBytecode()`.

**Comment by 0pdd** [comment]
@volodya-lombrozo 6 puzzles [#110](https://github.com/objectionary/jeo-maven-plugin/issues/110), [#111](https://github.com/objectionary/jeo-maven-plugin/issues/111), [#112](https://github.com/objectionary/jeo-maven-plugin/issues/112), [#113](https://github.com/objectionary/jeo-maven-plugin/issues/113), [#114](https://github.com/objectionary/jeo-maven-plugin/issues/114), [#115](https://github.com/objectionary/jeo-maven-plugin/issues/115) are still not solved.

**Comment by 0pdd** [comment]
@volodya-lombrozo 5 puzzles [#111](https://github.com/objectionary/jeo-maven-plugin/issues/111), [#112](https://github.com/objectionary/jeo-maven-plugin/issues/112), [#113](https://github.com/objectionary/jeo-maven-plugin/issues/113), [#114](https://github.com/objectionary/jeo-maven-plugin/issues/114), [#115](https://github.com/objectionary/jeo-maven-plugin/issues/115) are still not solved; solved: [#110](https://github.com/objectionary/jeo-maven-plugin/issues/110).

**Comment by 0pdd** [comment]
@volodya-lombrozo 4 puzzles [#111](https://github.com/objectionary/jeo-maven-plugin/issues/111), [#112](https://github.com/objectionary/jeo-maven-plugin/issues/112), [#114](https://github.com/objectionary/jeo-maven-plugin/issues/114), [#115](https://github.com/objectionary/jeo-maven-plugin/issues/115) are still not solved; solved: [#110](https://github.com/objectionary/jeo-maven-plugin/issues/110), [#113](https://github.com/objectionary/jeo-maven-plugin/issues/113).

**Comment by 0pdd** [comment]
@volodya-lombrozo 3 puzzles [#111](https://github.com/objectionary/jeo-maven-plugin/issues/111), [#112](https://github.com/objectionary/jeo-maven-plugin/issues/112), [#115](https://github.com/objectionary/jeo-maven-plugin/issues/115) are still not solved; solved: [#110](https://github.com/objectionary/jeo-maven-plugin/issues/110), [#113](https://github.com/objectionary/jeo-maven-plugin/issues/113), [#114](https://github.com/objectionary/jeo-maven-plugin/issues/114).

**Comment by 0pdd** [comment]
@volodya-lombrozo 3 puzzles [#111](https://github.com/objectionary/jeo-maven-plugin/issues/111), [#112](https://github.com/objectionary/jeo-maven-plugin/issues/112), [#122](https://github.com/objectionary/jeo-maven-plugin/issues/122) are still not solved; solved: [#110](https://github.com/objectionary/jeo-maven-plugin/issues/110), [#113](https://github.com/objectionary/jeo-maven-plugin/issues/113), [#114](https://github.com/objectionary/jeo-maven-plugin/issues/114), [#115](https://github.com/objectionary/jeo-maven-plugin/issues/115).

**Comment by 0pdd** [comment]
@volodya-lombrozo 3 puzzles [#111](https://github.com/objectionary/jeo-maven-plugin/issues/111), [#112](https://github.com/objectionary/jeo-maven-plugin/issues/112), [#124](https://github.com/objectionary/jeo-maven-plugin/issues/124) are still not solved; solved: [#110](https://github.com/objectionary/jeo-maven-plugin/issues/110), [#113](https://github.com/objectionary/jeo-maven-plugin/issues/113), [#114](https://github.com/objectionary/jeo-maven-plugin/issues/114), [#115](https://github.com/objectionary/jeo-maven-plugin/issues/115), [#122](https://github.com/objectionary/jeo-maven-plugin/issues/122).

**Comment by 0pdd** [comment]
@volodya-lombrozo 2 puzzles [#111](https://github.com/objectionary/jeo-maven-plugin/issues/111), [#112](https://github.com/objectionary/jeo-maven-plugin/issues/112) are still not solved; solved: [#110](https://github.com/objectionary/jeo-maven-plugin/issues/110), [#113](https://github.com/objectionary/jeo-maven-plugin/issues/113), [#114](https://github.com/objectionary/jeo-maven-plugin/issues/114), [#115](https://github.com/objectionary/jeo-maven-plugin/issues/115), [#122](https://github.com/objectionary/jeo-maven-plugin/issues/122), [#124](https://github.com/objectionary/jeo-maven-plugin/issues/124).

**Comment by 0pdd** [comment]
@volodya-lombrozo the puzzle [#112](https://github.com/objectionary/jeo-maven-plugin/issues/112) is still not solved; solved: [#110](https://github.com/objectionary/jeo-maven-plugin/issues/110), [#111](https://github.com/objectionary/jeo-maven-plugin/issues/111), [#113](https://github.com/objectionary/jeo-maven-plugin/issues/113), [#114](https://github.com/objectionary/jeo-maven-plugin/issues/114), [#115](https://github.com/objectionary/jeo-maven-plugin/issues/115), [#122](https://github.com/objectionary/jeo-maven-plugin/issues/122), [#124](https://github.com/objectionary/jeo-maven-plugin/issues/124), [#128](https://github.com/objectionary/jeo-maven-plugin/issues/128), [#129](https://github.com/objectionary/jeo-maven-plugin/issues/129).

**Comment by 0pdd** [comment]
@volodya-lombrozo all 10 puzzles are solved here: [#110](https://github.com/objectionary/jeo-maven-plugin/issues/110), [#111](https://github.com/objectionary/jeo-maven-plugin/issues/111), [#112](https://github.com/objectionary/jeo-maven-plugin/issues/112), [#113](https://github.com/objectionary/jeo-maven-plugin/issues/113), [#114](https://github.com/objectionary/jeo-maven-plugin/issues/114), [#115](https://github.com/objectionary/jeo-maven-plugin/issues/115), [#122](https://github.com/objectionary/jeo-maven-plugin/issues/122), [#124](https://github.com/objectionary/jeo-maven-plugin/issues/124), [#128](https://github.com/objectionary/jeo-maven-plugin/issues/128), [#129](https://github.com/objectionary/jeo-maven-plugin/issues/129).
